index.html

```diff
@@ -63,14 +63,22 @@
 
       </header>
       <div class="options shadow-sm rounded">
-        <i><span id="icon-palette" class="material-icons md-36 md-dark" title="Generate">palette</span></i>
+        <span id="generate-info">Generate palettes either by pressing <strong>spacebar</strong> or <strong>generate</strong> button!</span>
+        <!-- <i><span id="icon-palette" class="material-icons md-36 md-dark" title="Generate">palette</span></i>
         <i><span id="icon-undo" class="material-icons md-36 md-dark" title="Undo">undo</span></i>
         <i><span id="icon-redo" class="material-icons md-36 md-dark" title="Redo">redo</span></i>
-        
+         -->
         
```

Comment:
Kullanilmayan kod? copluk mu bura?

Reply:
Belki sonra kullanılır diye. Ama pr'a koymam bir daha töbe.